main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -119,6 +126,68 @@ Refine.OpenProjectUI._filterTags = function(tag) {
   });
 };
 
+Refine.OpenProjectUI.prototype._searchAnimation = function() {
+  const search = $('#searchIcon');
+  const form = $('.header-search-box');
+  const icon = $('.magnifying_glass');
+  search.click(function () {
+    if (form.is(':hidden')) {
+      $("#tagsUl").hide();
+      $("#divInput").show();
+      icon.addClass("magnifying-glass-open");
+      form.show();
+      form.focus();
+    }
+    const widthFormOpen = Math.floor($('#right-panel-body').width() * 4 / 5);
+
+    form.animate({
```

Comment:
I would remove this animation, I don't think it brings much and it does not feel consistent with what we do in the rest of the UI.

Reply:
Done